### PR TITLE
Feature/418

### PR DESCRIPTION
**Pull request by jrmartin**
Adds ID to password-reset button. Adds jquery to base-template.html, since other templates using the base-template.html require jquery, gets rid of console errors while user login and sign up. Re-arranges import of base-template in other templates to be the first thing that's loaded, since subsequent imports need jquery loaded first.

### CODE REVIEW

File: service/docker/Dockerfile-scidash [hunk 1]
```diff
@@ -9,9 +9,9 @@ ARG NRN_SYMLINK=/Applications/NEURON-7.6/nrn/
 ARG DOTENV_FILE=env-docker
 ARG STATIC_DIR=$APP_DIR/scidash/static
 ARG GEPPETTO_DIR=$STATIC_DIR/org.geppetto.frontend/src/main/webapp
-ARG targetBranch=geppetto-scidash
-ARG originBranch=geppetto-scidash
-ARG defaultBranch=geppetto-scidash
+ARG targetBranch=feature/418
```
Comment (ddelpiano) [inline service/docker/Dockerfile-scidash]: @jrmartin could you revert these changes? these should not be committed I think.

File: service/scripts/install-frontend.sh [hunk 2]
```diff
@@ -5,9 +5,9 @@ node -v
 root_path=$PWD;
 
 geppetto_repo="https://github.com/openworm/org.geppetto.frontend.git"
-geppetto_branch="geppett-scidash"
+geppetto_branch="scidash-jest-changes"
```
Comment (ddelpiano) [inline service/scripts/install-frontend.sh]: @jrmartin same here

**Review by ddelpiano — CHANGES_REQUESTED** [review]
@jrmartin this is good for me, I just need a clean-up of the branches you hardcoded (I guess to build your local env). Thx!

**Comment by jrmartin** [comment]
@ddelpiano Hard coded paths removed

**Review by ddelpiano — APPROVED** [review]

**Pull request merged**